packages/client/src/client/App/Trades/TradesGrid/styled.ts

```diff
@@ -9,7 +9,6 @@ import { LimitCheckStatus, TradeStatus } from "services/trades/types"
 import styled, { css } from "styled-components"
 
 export const Table = styled.div`
-  border-bottom: 15px solid ${({ theme }) => theme.core.darkBackground};
```

Comment:
this came from LC table mods .. is the LC instance still ok?

Reply:
oh, looks like this was just moved around, not introduced for LC .. still worth checking all is well, tho

Reply:
LC still looks okay, but a good idea to check